scripts/python/homologous/in_solution.py

```diff
@@ -0,0 +1,137 @@
+# Code generates solution for homologous collpasing core (Goldreich & Weber). 
+
+import numpy as np
+import math
+import time
+import matplotlib.pyplot as pl
+import sys
+import glob
+from scipy.integrate import odeint
+
+
+G=6.69e-8
+c=3.e10
+msun=2.e33
```

Comment:
Units/constants package? Also you should probably use higher precision for these numbers.